frontend/src/components/FormFooter.jsx

```diff
@@ -0,0 +1,61 @@
+import React from "react";
```

Comment:
Please add `// NPM Packages` above import React and an empty line after the import